### ISSUE TITLE
Maintenance status?

### ISSUE THREAD

**Issue by Vacilando** [jessestuart/gatsby-source-s3-image#355]
We see some crucial problems open for a long time and lots of PR's. 

Is this plugin still being maintained?

**Comment by tomekrozalski** [comment]
Good question. `gatsby-source-s3-image` seems to not work with Gatsby v2. `gatsby-source-s3` works, but is not so great, because it doesn't work so convenient with `gatsby-image`. If I am wrong, please correct me :)

**Comment by good3n** [comment]
> Good question. `gatsby-source-s3-image` seems to not work with Gatsby v2. `gatsby-source-s3` works, but is not so great, because it doesn't work so convenient with `gatsby-image`. If I am wrong, please correct me :)

That is correct. I also checked out `gatsby-source-s3` and it appears that the ability to use it with `gatsby-image` has broken. Bummer.

**Comment by robinmetral** [comment]
FYI, after waiting for maintenance updates from both `gatsby-source-s3-image` and `gatsby-source-s3`, I ended up making my own version in TypeScript at https://github.com/robinmetral/gatsby-source-s3.

Feel free to use it if it helps! Any contribution over there is more than welcome, including any bug reports or feature requests 🙂

I intend to actively support it for at least as long as this plugin's maintenance status is unclear.

**Comment by patspam** [comment]
I ended up switching to `@robinmetral/gatsby-source-s3` after trying the other versions.
